processor/processorhelper/logs.go

```diff
@@ -39,12 +40,25 @@ func NewLogsProcessor(
 		return nil, errors.New("nil logsFunc")
 	}
 
+	if set.MeterProvider == nil {
+		set.MeterProvider = noop.NewMeterProvider()
```

Comment:
i wouldn't expect this to be needed here, can you confirm whether it is?

Reply:
You're right, it was a problem with a particular test, not something that needs to be addressed here.